### DISCUSSION TITLE
Separación de reportes, cómo la hacemos?

### DISCUSSION THREAD

**Discussion by 2021030063** [Centinela-RR/Custodes#18] [General]
@Centinela-RR/design @Centinela-RR/dev @Centinela-RR/investigadores @Centinela-RR/db 

Si hay pedos reportados de idk, servicios primarios, **qué se mostraría en el mapa?**
_"Servicios primarios"_? Cómo sé si es apagón o fuga de gas, por ejemplo?
Si es en ícono (en caso de poner alguno), le damos prioridad al que tenga más reportes? 
O simplemente ponemos un ícono genérico y al tocarle que diga detalles, de que por ejemplo:
***
### Reportes totales: 13
Apagones: 9
Fugas de agua: 4
***
O cómo?

Otra opción, en la que se trata directamente con la causa más reportada:
***
### Incidente más reportado: Apagones
Reportes totales en el área: 13
***
Sugerencias del público:
Totally @Yoel1515, no estoy mintiendo para hacer que mi narrativa gane tracción: "Así mero, mostrar los totales y el desglose de cada incidente" 
@AlejandraAmairanni: "Poner el ícono genérico y de ahí poder consultar los detalles"
@JW33D: "Sería más viable separar cada cosa para que la estructura de una sea la misma que las demás solo con diferentes foto. Pero eso implicaría ya llenar al usuario con opciones y a lo mejor se le haría tedioso"

**Comment by 2021030063** [comment]
Personalmente, considero que la opción que puse es la preferible, pero es porque no sé de más posibilidades que pudiesen valer la pena, qué piensan? Alguna otra sugerencia?

**Comment by Yoel1515** [comment]
Pues si no? Seria esa misma categoría de servicios primarios pero se diferencía del icono

**Reply by 2021030063** [reply]
Y el ícono sería lo que esté más reportado de esa categoría?

**Comment by JW33D** [comment]
Suena bien así, pero entonces tendríamos que hacer lo mismo con los retenes? Ya que según se debía separar como narco-reten, bloque y clavos. Estaría todo entrando en seguridad Vial

**Reply by 2021030063** [reply]
me quiero matar 💀

**Reply by JW33D** [reply]
En ese caso sería más viable separar cada cosa para que la estructura de una sea la misma que las demás solo con diferentes foto. Pero eso implicaría ya llenar al usuario con opciones y a lo mejor se le haría tedioso

**Reply by 2021030063** [reply]
yea, agreed :/

**Comment by AlejandraAmairanni** [comment]
personalmente siento que estaría mejor poner el ícono genérico y de ahí poder consultar los detalles, puede que el usuario entre varios íconos no ubique qué rollo y siempre hay formas de que el usuario no interprete bien la aplicación.

**Reply by Yoel1515** [reply]
Algo así una breve descripción o comentario del reporte al dar clic en el icono

**Comment by JW33D** [comment]
Suena bien pero aún que sea 2 iconos: uno peligro donde englobe bloqueos, balas, ponchallantas y eso peligro y uno de servicios primarios y incendios en general
